### PR TITLE
D2-1399 - Adapt to core v5

### PR DESCRIPTION
**Pull request by willianoki**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1 +1,11 @@
...
+.eslint*
+.mocharc.json
+.nycrc
 .nyc_output
+.vscode
+.idea
+coverage
+scratches
+spec
+src
```
Comment (wesleysmyth) [inline .npmignore:11]: Ignore src?

Comment (willianoki) [reply .npmignore:11]: @wesleysmyth I guess we don't need src, just dist, in the package; do you think we should keep packaging the src?

**Review by joseflexadaitan — APPROVED** [review]
LGTM

File: .npmignore (hunk 1 above)
Comment (wesleysmyth) [reply .npmignore:11]: Ahh this is fine, thought this was the .gitignore 🤦

**Review by wesleysmyth — APPROVED** [review]

**Pull request merged**